packages/manfred2html/src/engine/cv-jedi-minimalism/html-parts/contact-section/contact-section.ejs

```diff
@@ -0,0 +1,98 @@
+<%_ if (profile.relevantLinks && profile.relevantLinks !== 0) { -%>
+  
+  <h2>Contact</h2>
+      <div class="aside__container">     
+       <%_ for (const link of profile?.relevantLinks) { -%>
+        <%_ if (link.type === 'phone') { -%>
+          <div class="aside__container__item">
+            <div>
+              <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path
+                  d="M1.5 2.3125C1.5 1.86377 1.86377 1.5 2.3125 1.5H4.06171C4.45889 1.5 4.79786 1.78715 4.86315 2.17893L5.46385 5.78307C5.52249 6.13491 5.3448 6.48385 5.02576 6.64337L3.7679 7.2723C4.67489 9.52614 6.47387 11.3251 8.7277 12.2321L9.35663 10.9742C9.51615 10.6552 9.86509 10.4775 10.2169 10.5362L13.8211 11.1368C14.2129 11.2021 14.5 11.5411 14.5 11.9383V13.6875C14.5 14.1362 14.1362 14.5 13.6875 14.5H12.0625C6.22899 14.5 1.5 9.77101 1.5 3.9375V2.3125Z"
+                  fill="#DAD6D1"
+                />
+              </svg>
+            </div>
+            <a href="tel:+#">+1 041 234 5678</a>
```

Comment:
If we have phone display phone. Add a condition and use props to display phone

Reply:
replace by <%=link.URL%>